sample_inventories/kafka_connect_replicator.yml

```
@@ -0,0 +1,69 @@
+---
+### Kafka Connect Replicator - Configuration Example with Security
+##
+## The following is an example inventory file of the configuration required for setting up Kafka Connect Replicator between two
+## clusters with TLS and SASL Plain on the source cluster and TLS and Kerberos on the Destination cluster.
+
+all:
+  vars:
+
+kafka_connect_replicator:
+
+  kafka_connect_replicator_listener:
+  ssl_enabled: true
+  ssl_mutual_auth_enabled: false
+  sasl_protocol: kerberos
```

Shouldn't this be like this?
```suggestion
  kafka_connect_replicator_listener:
    ssl_enabled: true
    ssl_mutual_auth_enabled: false
    sasl_protocol: kerberos
```